src/zamp_converter/zcl_amp_conv_http_json.clas.abap

```diff
@@ -8,53 +8,51 @@ CLASS zcl_amp_conv_http_json DEFINITION
     INTERFACES zif_amp_converter .
   PROTECTED SECTION.
   PRIVATE SECTION.
+
+    METHODS escape_json
+      IMPORTING i_val        TYPE string
+      RETURNING VALUE(r_val) TYPE string.
```

Comment:
@ThoralfHaensel could you remove the prefixes?
We don't use them 😄

Reply:
@JohannesKonings Done